test/sequential/bootstrap.cancelBootstrap.test.ts

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable jest/no-conditional-expect */
+/*
+ *
+ * @group bootstrap
+ * @group sequential
+ */
+import { getApi, initApi } from "../../utils/api";
+import {
+  scheduleBootstrap,
+  finalizeBootstrap,
+  cancelRunningBootstrap,
+} from "../../utils/tx";
+import { EventResult, ExtrinsicResult } from "../../utils/eventListeners";
+import { Keyring } from "@polkadot/api";
+import { User } from "../../utils/User";
+import {
+  getEnvironmentRequiredVars,
+  waitForBootstrapStatus,
+} from "../../utils/utils";
+import {
+  getEventResultFromMangataTx,
+  sudoIssueAsset,
+} from "../../utils/txHandler";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { BN, MangataGenericEvent, toBN } from "@mangata-finance/sdk";
+import { hexToU8a } from "@polkadot/util";
+import { Assets } from "../../utils/Assets";
+import { Sudo } from "../../utils/sudo";
+import { setupApi, setupUsers } from "../../utils/setup";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.spyOn(console, "error").mockImplementation(jest.fn());
+jest.setTimeout(3500000);
+process.env.NODE_ENV = "test";
+
+let testUser1: User;
+let sudo: User;
+let keyring: Keyring;
+let bootstrapPhase: any;
+let bootstrapCurrency: any;
+let eventResponse: EventResult;
+
+const { sudo: sudoUserName } = getEnvironmentRequiredVars();
+const waitingPeriod1 = 400;
+const waitingPeriod2 = 15;
```

Comment:
instead of ' waitingPeriod1' and ' waitingPeriod2' rename them to something meaninful please. Its difficult to understand the tests by reading them.